modules/monitoring-accessing-the-metrics-targets-page.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * monitoring/managing-metrics-targets.adoc
+
+[id="monitoring-accessing-the-metrics-targets-page_{context}"]
+= Accessing the Metrics Targets page in the Administrator perspective
+
+You can view the *Metrics Targets* page in the Administrator perspective in the {product-title} web console.
+
+.Prerequisites
+
+* You have access to the cluster as an administrator for the project for which you want to view metrics targets.
+
+.Procedure
+
+* In the *Administrator* perspective, select *Observe* -> *Targets*. 
```

Comment:
IBM SG 161: `>` not `->` So, `...select *Observe* > *Targets*.`

Reply:
That's an OpenShift docs exception to the IBM SG: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference